skbio/io/format/tests/test_stockholm.py

```diff
@@ -245,6 +250,14 @@ def test_multiple_msa_file(self):
                          index=['RTC2231', 'RTF2124', 'RTH3322', 'RTB1512'])
         self.assertEqual(msa, exp)
 
+    def test_stockholm_maintains_order(self):
+        fp = get_data_path('stockholm_metadata_only')
```

Comment:
Can you test that order is maintained for all four types of metadata? This is just testing GF.